### PR TITLE
Adding zappy pools and strategies

### PR DESCRIPTION
**Pull request by jokbal-yum**
-Adding zappy pools and strategies
-Fixing decimals for WBTC from 18 to 8
-Added zappy interface to address book
-Updated address book to 0.5.0
-Added APY calculation for zappy

CleanShot 2022-04-05 at 13 42 06@2x

### CODE REVIEW

File: packages/address-book/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@yieldhubfinance/blockchain-addressbook",
-  "version": "0.4.0",
+  "version": "0.5.0",
```
Comment (yieldrancher) [inline packages/address-book/package.json:3]: We should decide on a versioning that isn't a major bump for every change, but LGTM for now

File: src/api/stats/getAmmPrices.ts [hunk 2]
```diff
@@ -3,15 +3,14 @@
...
 const INIT_DELAY = 0 * 60 * 1000;
 const REFRESH_INTERVAL = 5 * 60 * 1000;
 
 // FIXME: if this list grows too big we might hit the ratelimit on initialization everytime
-// Implement in case of emergency 
-const pools = [
-  ...omnidexPools,
-];
+// Implement in case of emergency
+const pools = [...omnidexPools, ...zappyPools];
```
Comment (yieldrancher) [inline src/api/stats/getAmmPrices.ts]: Keep this on multiple lines so the diff is more obvious

Comment (yieldrancher) [inline src/api/stats/getAmmPrices.ts]: Keep this on multiple lines so the diff is more obvious

File: src/api/stats/telos/index.js [hunk 3]
```diff
@@ -1,8 +1,7 @@
 const getOmnidexLpApys = require('./getOmnidexLpApys');
+const getZappyLpApys = require('./getZappyLpApys');
 
-const getApys = [
-  getOmnidexLpApys,
-];
+const getApys = [getOmnidexLpApys, getZappyLpApys];
```
Comment (yieldrancher) [inline src/api/stats/telos/index.js]: Multiline

Comment (yieldrancher) [inline src/api/stats/telos/index.js]: Multiline

Comment (yieldrancher) [inline src/api/stats/telos/index.js]: Multiline

Comment (yieldrancher) [inline src/api/stats/telos/index.js]: Multiline

Comment (yieldrancher) [inline src/api/stats/telos/index.js]: Multiline

**Review by yieldrancher — APPROVED** [review]
LGTM with a couple of nits

File: packages/address-book/package.json (hunk 1 above)
Comment (jokbal-yum) [reply packages/address-book/package.json:3]: ah yeah.. i thought this was already minor version since i thought first one would be major but you're right

**Comment by jokbal-yum** [comment]
Comments resolved ,merging

**Pull request merged**